packages/InteractiveProfilingTool-Tests.package/IPTStatsTestCase.class/instance/testFormatLeavesWithoutLeaves.st

```diff
@@ -0,0 +1,10 @@
+tests
+testFormatLeavesWithoutLeaves
+
+	| formatted message |
+
+	(model userCallsRoots at: 1) receivers: {}.
+	formatted := model formatLeaves.
+	message := 'There are no leaves relevant to runtime performance.'.
```

Comment:
Sollten wir hier ggf. die entsprechende Methode statt der konkreten Nachricht aufrufen? Wenn die Meldung geändert wird (siehe mein Vorschlag dazu) muss das an zwei Stellen geändert werden -> Code Smell :see_no_evil: 

Der Punkt am Ende kann weg

Reply:
> Wenn die Meldung geändert wird (siehe mein Vorschlag dazu) muss das an zwei Stellen geändert werden -> Code Smell

Zum Hintergrund: In Tests bin ich mir da nicht so sicher – und ich glaube es gibt da ähnliche starke Meinungen zu wie zu Tabs vs Spaces.

Würde hier die Konstante anstatt eines Literals verwendet werden, wäre der Test selbst von dem System abhängig, das er testen soll. Er hängt dann nicht nur von der erwarteten Nachricht ab, sondern auch von Implementierungsdetails wie der Tatsache das überhaupt eine Konstante verwendet wird und davon, wie die Konstante benannt ist.

Sollte die Nachricht tatsächlich mal geändert, wird einmal ein Fehler geworfen, der Test angepasst. Das dient gleichzeitig als Bestätigung, dass die Nachricht bewusst geändert wurde.

Wir haben aber anderer Stelle auch schon Konstanten verwendet z. B. `IPTReport >> highRuntimeIcon`, weil es sonst einfach extrem hässlich geworden wäre. Bin da also grundsätzlich auch nicht so ganz entschieden.

Reply:
Fehlermeldung ist irrelevant, s. o.